src/main/java/seedu/address/model/person/insurance/claim/Claim.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.model.person.insurance.claim;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * An object that represents a claim that the client has made under an insurance plan.
+ */
+public class Claim {
+    public static final String INVALID_CLAIM_ID = "Invalid Claim ID, must follow: "
+            + "Letter + 4 digits only.";
+
+    public static final String NEGATIVE_CLAIM_AMOUNT = "Claim Amount cannot be negative";
+    public static final String INVALID_CLAIM_AMOUNT = "Claim Amount must be a positive integer";
+
+    private final String claimId;
+    private final boolean isOpen;
+    private final int claimAmount;
+
+    /**
+     * Creates a new open Claim based on claim id and amount. This constructor is used when user creates a new claim.
+     *
+     * @param claimId     claim id obtained through official channels.
+     * @param claimAmount claim amount recorded in cents, can only be a positive integer.
+     */
+    public Claim(String claimId, int claimAmount) {
+        this.claimId = claimId;
+        this.isOpen = false;
```

Comment:
Shouldn't it be isOpen = true when claim is created? Then it will be isOpen = false when the claim is closed

Reply:
Yup, agree

Reply:
yep good catch!